### PR TITLE
[14.0][MIG] currency_rate_inverted: Migration to 14.0

### PR DESCRIPTION
**Pull request by tupaq**

### CODE REVIEW

File: currency_rate_inverted/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 {
     "name": "Currency Rate Inverted",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.0",
```
Comment (alexey-pelykh) [inline currency_rate_inverted/__manifest__.py:9]: This version change should be done as a separate commit. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: currency_rate_inverted/__manifest__.py (hunk 1 above)
Comment (tupaq) [reply currency_rate_inverted/__manifest__.py:9]: I already corrected

**Comment by SkiBY** [comment]
As behavior is the same in v.14, you can get changes from https://github.com/OCA/currency/pull/107 directly to this PR

**Comment by MiquelRForgeFlow** [comment]
This should be closed, as the module was already merged in https://github.com/OCA/currency/pull/108.
